docker/spec/dependabot/docker/utils/helpers_spec.rb

```diff
@@ -0,0 +1,33 @@
+# # frozen_string_literal: true
+
+require "dependabot/dependency_file"
+require "dependabot/docker/utils/helpers"
+require "spec_helper"
+
+RSpec.describe Dependabot::Docker::Utils.likely_helm_chart? do
```

Comment:
Since this is a static method on the class, not sure that this is the proper way to call it? Writing this was a reminder I _really_ need to spend some time on basic ruby syntax tutorials... 😀